src/main/webapp/resources/js/viewer/viewer.js

```diff
@@ -474,6 +474,41 @@ class Viewer {
 		this._currentZoom = zoomfactor;
 	}
 
+	rotateImage(orientation, point) {
+		if (point == null) {
+			point = this._image.bounds.center;
+		}
+		this._image.rotate(orientation, point);
+		/*		let newBounds = this._image.bounds.clone();
+
+                let trueCenter = new paper.Point(newBounds.topLeft.x + (newBounds.topRight.x - newBounds.topLeft.x)/2, newBounds.topLeft.y + (newBounds.bottomLeft.y - newBounds.topLeft.y)/2);
+                return trueCenter;*/

```

Comment:
Can the commented out code be deleted or is it useful for debugging or sth else?

Reply:
Can be deleted.